src/helpers/graph.js

```diff
@@ -0,0 +1,22 @@
+export function addGroupTagToNotes(notes, edges) {
```

Comment:
It's a bit weird that in the graph helper we work with `noTes`, and not with `noDes`.